apps/web/e2e/tests/connectWallet.spec.ts

```diff
@@ -0,0 +1,12 @@
+import { test } from '../testFixture';
+import { connectWallet } from '../appSession';
+
+test.describe('Connect Wallet', () => {
+  test('should connect to wallet', async ({ page, metamask }) => {
+    if (!metamask) {
+      throw new Error('Metamask is not defined');
+    }
```

Comment:
Does every test we write need this? Is there a way we can make onchaintestkit perform this check for us?